server/resolvers/book/mutation.js

```diff
@@ -0,0 +1,72 @@
+const { Author, Book } = require("../../models");
+const { Metadata, Review } = require("../../models/mongodb");
+const { sequelize } = require("../../models");
+
+const BookMutation = {
+  addBook: async ({ title, description, published_date, author_id }) => {
+    const sequelizeTransaction = await sequelize.transaction();
```

Comment:
anyway to make the transaction, commit, rollover part common?

Reply:
Yes, we can make this a utility.